src/components/DataBar/DataBar.module.css

```diff
@@ -8,15 +8,18 @@
  * The data bar represents the relative completion of a task.
  */
 .data-bar {
-  --data-bar-height: var(--eds-size-2);
+  --data-bar-height: 0.875rem;
```

Comment:
We should validate with design the spacing between things. I've seen a few cases where the spacing used by design doesn't match these `eds-size` values. If we have tokens, we should stick with them!

Reply:
This is 14px and I think this makes sense since the data bar was made to be 12px, with a 1px border.
Also is the same height as the progress bar.
However will check with design to be sure

Reply:
ah ok. so it's 12px + borders on both sides? in that case, I think we are good, and this value could be a calc of the appropriate tokens (eds-size-1-and-half? and 2x border size or similar.

We wouldn't want the height to break if either of these tokens change

Reply:
Could make it a calc to make it more obvious yea